src/factories/border.ts

```diff
@@ -8,6 +8,7 @@ export type BorderStyle = {
   stroke: number,
   radius?: number,
   color?: ColorSource,
+  roundedTop?: boolean,
```

Comment:
Thoughts on adding `topLeftRadius`, `bottomLeftRadius`, etc rather than adding some specific to this implementation?

Reply:
Yeah I almost did like `radius: number | [number, number, number, number]`. I still could. Just felt like the need to have one radius different would be pretty unlikely to happen for the effort. (have to create separate calls for corners and all that)

Reply:
seems likely that we might need a "roundedBottom" or something though. `roundedTop` seems overly specific to the currently implementation to belong in something that's generic to creating borders

Reply:
Updated to support `radius: number | [number, number, number, number]`